packages/common/src/components/AppHeader/AppHeaderTabs.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { UserTab } from '../../types/utils'
+import { CustomButton } from '@frontend/common'
+import styled from 'styled-components'
+import { useHistory, useRouteMatch } from 'react-router-dom'
+
+const Wrapper = styled.nav`
+  border: 1.5px solid rgba(1, 165, 163, 0.08);
```

Comment:
Don't we have predefined constants for the colors we use?

Reply:
We tried using the css function var() before to access the colors. It worked locally but in the test env it didn't find the colors. Is there another way to do it?

Reply:
You could just define them in javascript, that's better than reproducing the values all over the code base.

Reply:
Fixed

Reply:
Fixed